### ISSUE TITLE
Handle cases where multiple authors have been linked with same name

### ISSUE THREAD

**Issue by arthurpsmith** [arthurpsmith/author-disambiguator#12]
Example: https://www.wikidata.org/wiki/Q56917927 - Wang Jun/Jun Wang

**Comment by arthurpsmith** [comment]
Bringing up the author page for Q16027697 shows a ton more examples! Looking into this...

**Comment by arthurpsmith** [comment]
I've added a form to the author items page to select papers for which you can remove the P50 statements, or move them to another author. Tested and it seems to work... This is still not a complete solution, but it at least helps address the problem when the assignments are clearly wrong.

**Comment by arthurpsmith** [comment]
I did a bunch of cleanup on Wang Jun/Jun Wang; the vast majority of the papers seem to have been associated with a single researcher (an administrator at the Beijing Genome Institute also associated with various other places). The tools I've added helped a lot in moving items to the right author and cleaning up the cases with two linked authors. I think this is sufficient at least for now - closing.
